x-pack/plugins/security_solution/server/endpoint/lib/metadata/check_metadata_transforms_task.ts

```diff
@@ -118,41 +124,53 @@ export class CheckMetadataTransformsTask {
     }
 
     const { transforms } = transformStatsResponse.body;
-    if (!transforms.length) {
+    let { reinstallAttempts } = taskInstance.state;
```

Comment:
I don't see the code below wrapped in a `try {}` block. Should it be? Since this is a task, I don't know what happens if it throws without us handling it. Would the task stop running?

Reply:
tasks definitely run in a different execution logic than http handlers. I don't know if throwing inside a running task will stop future ones (worth double-checking). But if it doesn't, then it's fine not to `try`, and let the next attempt come around